psh/psh.h

```diff
@@ -20,8 +20,8 @@
 
 typedef struct psh_app {
 	const char name[11];
-	const int (*run)(int argc, char **argv);
-	const void (*info)(void);
+	int (*run)(int argc, char **argv);
+	void (*info)(void);
```

Comment:
I guess this was the intention here
```suggestion
	int (*const run)(int argc, char **argv);
	void (*const info)(void);
```

Reply:
yeah, I didn't check the usage across the codebase, switched to the suggested type